mirrord-layer/src/lib.rs

```diff
@@ -425,11 +425,38 @@ fn enable_hooks() {
         )
         .unwrap();
 
-    interceptor
-        .replace(
-            Module::find_export_by_name(None, "accept").unwrap(),
-            NativePointer(accept_detour as *mut c_void),
-            NativePointer(std::ptr::null_mut::<c_void>()),
-        )
-        .unwrap();
+    match Module::find_export_by_name(None, "uv__accept4") {
```

Comment:
we don't want to hook the other functions if this is none, rather hook each one of those if not hooked already.